samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs

```diff
@@ -151,14 +150,11 @@ private void RemoveArcGISMapView()
         }
     }
 
-    private void SceneChanged()
+    // Stop the menu background video from player as soon as other scene is loaded
```

Comment:
In general, I would get rid of all the function comments across the whole code, but only if its not educational comments
```suggestion
```